utils/crates-io/src/version.rs

```diff
@@ -33,17 +33,26 @@ struct Version {
 }
 
 /// Verify if the package has already been published.
-pub fn verify(name: &str, version: &str) -> Result<bool> {
+pub fn verify(mut name: &str, version: &str) -> Result<bool> {
+    println!("Verifying {}@{} ...", &name, &version);
+
+    if name == "gear-core-processor" {
```

Comment:
maybe go through all aliases finding match? here and in other places

Reply:
We have all of our packages names on crates-io except `gear-core-processor`, see https://crates.io/crates/gear-core-processor, it is taken by others, I'll gather this func in `handler.rs` and gather the docs of `core-processor`